R/check_filename.R

```diff
@@ -1,4 +1,38 @@
-#' Check the style of file names
+#' Check the style of file and folder names
+#'
+#' A consistent naming schema avoids problems when working together.
+#' Especially when working with different OS.
```

Comment:
The second sentence has no verb and is as such not a sentence, so I suggest adding it to the previous sentence.
```suggestion
#' A consistent naming schema avoids problems when working together, especially when working with different OS.
```